src/views/EditFolder/actions/EditFolder.actions.ts

```diff
@@ -157,8 +157,8 @@ export const createNewFolder: CreateNewFolderFn = folderName => dispatch => {
         hashHistory.push('/');
     })
     .catch(error => {
-        dispatch(handleFailedFolderCreation(error));
-
-        hashHistory.push('/');
+        dispatch(handleFailedFolderCreation('sorry'));
```

Comment:
Hardcoded result?
If you want to show the same hardcoded message for every case, just return action with type only (no payload) and hardcode the message in the reducer.